rialto_airflow/publish/publication_utils.py

```diff
@@ -26,7 +26,8 @@ def _openalex_journal(openalex_json):
     try:
         primary_location = openalex_json["primary_location"]
```

Comment:
sometimes primary_location in OpenAlex data is None -- so we need to guard against that.